### PR TITLE
Suggestions RB

### PR DESCRIPTION
**Pull request by RobbinBouwmeester**

### CODE REVIEW

**Comment by RobbinBouwmeester** [comment]
I do have some worries about how the terms "tool", "algorithm", "model", and "software" are used. I think in addition to some of the comments it would be good that we agree on the semantics.

**Comment by bittremieux** [comment]
> Different to other fields in bioinformatics, proteomics software tools and algorithms often use custom licenses, with complex clauses and terms that confuse the users and developers. [really is this common practice? I personally know of only a single project]

I agree with Robbin that I'd be surprised if our community would really be so much worse than others. I don't have hard data, but I think these are universal challenges.

File: manuscript.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ OSS (open-source software) refers to software where the source code—the core i
...
+A common misconception is that closed-source software with "academic" licenses are equivalent to open-source. Adding to the confusion, some journals now require code availability as a condition of publication. This has led researchers to upload their closed-source software to collaborative platforms. While this provides the appearance of openness through features like issue trackers, the source code remains locked away. Even if well-intentioned, this can be misleading to other scientists and in our view discouraged. [what about github repos starting as open-source, changing over time to be closed source under the same name and caursing confusion. Or being open-source at the time of review and removing repositories. I think this should maybe be mentioned here? On second though, probably not here, but I think we should make this point later on.]
```
Comment (ypriverol) [inline manuscript.md]: @RobbinBouwmeester I agree with you, that is another problem, can you highlight that? It is true that some software starts from open-source and ends up in a company with the same software through a route of close, source, change license, and finally making the software commercial. 

Please feel free to include that as another point.

File: manuscript.md [hunk 2]
```diff
@@ -104,14 +104,14 @@ We have indeed observed this in our own projects: At the time of writing, QuantM
 
 Before we delve into the role of OSS in computational proteomics, and before losing the reader in the details of the manuscript and the importance of FAIR software, we would like to highlight one of the fundamental aspects and challenges in the development of proteomics software: the use of software licenses.
 
-Different to other fields in bioinformatics, proteomics software tools and algorithms are often using custom licenses, with complex clauses and terms that confuse the users and developers.
+Different to other fields in bioinformatics, proteomics software tools and algorithms often use custom licenses, with complex clauses and terms that confuse the users and developers. [really is this common practice? I personally know of only a single project]
```
Comment (ypriverol) [inline manuscript.md]: I can compile statistics, but I know from Bioconda that when we were dealing with proteomics tools, a lot of them were custom licenses. Instead, Sam tools, data, and other big projects in genomics use standard licences. I think @bgruening from Bioconda and Galaxy can help you figure this out. I will try to compile some stats if I find time.

**Review by ypriverol — CHANGES_REQUESTED** [review]

**Comment by ypriverol** [comment]
@RobbinBouwmeester I have added some requests and reviewed them. Can you let me know what you think? Please also in the section where you feel the text could be written more positively. Can you give it a try?

File: manuscript.md (hunk 2 above)
Comment (bgruening) [reply manuscript.md]: fragpipe, msfragger, DIA-NN, MaxQuant, everything that uses RawFileReader and a few more I removed from my memory. It's a huge problem in Proteomics compared to Genomics.

File: manuscript.md (hunk 1 above)
Comment (wfondrie) [reply manuscript.md]: I agree too - just be cautious not to call out specific examples here. We're after all not trying to shame existing tools, but rather help us build a better future 😄

**Comment by wfondrie** [comment]
> I do have some worries about how the terms "tool", "algorithm", "model", and "software" are used. I think in addition to some of the comments it would be good that we agree on the semantics.

I also agree with this! I was a bit sloppy in my writing when switching between the terms haphazardly.

**Comment by ypriverol** [comment]
@RobbinBouwmeester can you update your Fork with the current version? I accepted some minor changes from @bittremieux.

File: manuscript.md (hunk 2 above)
Comment (wfondrie) [reply manuscript.md]: See issue #8

**Review by wfondrie — COMMENTED** [review]
I like the changes, comments, and suggestions here. 👏 

Happy to approve once your changes are in @RobbinBouwmeester.

**Review by ypriverol — APPROVED** [review]

**Pull request merged**